.github/workflows/build.yml

```diff
@@ -14,15 +14,21 @@ jobs:
       fail-fast: false
       matrix:
         distribution:
-          - image: alpine
+          - image: gentoo-openrc
             version: "latest"
-            platform: "linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8"
+            platform: "linux/amd64"
           - image: gentoo
             version: "latest"
```

Comment:
Openrc is the default init system of Gentoo, You can replace gentoo-systemd image by Gentoo openrc. Just keep the version to latest.